index.js

```diff
@@ -1,7 +1,108 @@
 // Iteration 1: Names and Input
-
+const hacker1 = 'Lucas';
+console.log(`The driver's name is ${hacker1}`);
+const hacker2 = 'Maria';
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
+const longHacker1 = hacker1.length;
+const longHacker2 = hacker2.length;
 
+if (longHacker1 === longHacker2) {
+  console.log(`Wow, you both have equally long names, ${longHacker1} characters!.`);  
+} else if (longHacker1 > longHacker2) {
+  console.log(`The driver has the longest name, it has ${longHacker1} characters.`);
+} else {
+  console.log(`It seems that the navigator has the longest name, it has ${longHacker2} characters.`);
+}
 
 // Iteration 3: Loops
+//3.1
+const hacker1Upper = hacker1.toUpperCase();
+let stringResultHacker1 = ''
+for (let i = 0; i < longHacker1; i++) {
+  stringResultHacker1 += hacker1Upper[i];
+  if(i < (longHacker1 - 1)) {
+    stringResultHacker1 += ' ';
+  }
+}
+console.log(stringResultHacker1);
+
+//3.2
+let stringResultHacker2 = ''
+for (let i = longHacker2 - 1; i >= 0; i--) {
+  stringResultHacker2 += hacker2[i];
+}
+console.log(stringResultHacker2);
+
+//3.3
+let lexicographicResult = 0;
+
+
+if (hacker1 === hacker2) {
+  console.log('What?! You both have the same name?');
+} else if (hacker1[0] < hacker2[0]) {
```

Comment:
```suggestion
} else if (hacker1 < hacker2) {
```

No te haria falta mirar por la primera palabra, imaginate que son Carlos y Carla, si le quitas el 0 listo!

Reply:
Genial muchas gracias por la corrección!!! ya lo modifique en el código, quieres que vuelva a hacer la pull request?